scm/github.go

```diff
@@ -595,14 +581,14 @@ func (s *GithubSCM) createTeam(ctx context.Context, opt *TeamOptions) (*Team, er
 			Name: opt.TeamName,
 		})
 		if err != nil {
-			if opt.TeamName != TeachersTeam && opt.TeamName != StudentsTeam {
+			if opt.TeamName != TeachersTeam {
```

Comment:
I’m confused by this code. Why could it happen that the teacher team already exists, if the `team` variable is nil in the above check (Line 578)?

Also, on Line 572, we use `slug.Make` twice, but continue to use both variables being slugged without slugging…

Reply:
Yes, I also don't quite get it what and why is happening in this method. Should I create a new issue to rework it?

Reply:
Updated slugs. We really want to only use the slugified team name here, but I think it is unnecessary for the organization name.